e2e/page_objects/programCatalog.js

```diff
@@ -5,7 +5,14 @@ class ProgramCatalog {
     constructor() {
         this.logo = element(by.css('.header__navbar--logo'));
         this.searchBar = element(by.css('.adjust-search input'));
-        this.card = element.all(by.css('.card__inner'));
+        this.cards = element.all(by.css('.card__inner'));
+        this.selectedFiltersField = element.all(by.css('.filters'));
+        this.resultCounter = element(by.css('.result-count'));
+        this.dropdownSelector = text => element(by.cssContainingText('.multiselect-dropdown', text));
+        this.dropdownFilterSelector = text => element(by.cssContainingText('.multiselect-item-checkbox', text));
+        this.courseLevelLogo = card => card.element(by.css('span > .course-level'));
+        this.dropdownSelectorisVisible = element(by.css('.dropdown-list'));
```

Comment:
Please, change the name of this selector. It looks like a method name. It could be dropdownList instead.